test/fuzz/test_runner.py

```diff
@@ -259,9 +260,16 @@ def job(command, t, t_env):
     for target, t_env in targets:
         target_corpus_dir = corpus_dir / target
         os.makedirs(target_corpus_dir, exist_ok=True)
+        use_value_profile = int(random.random() < .3)
+        max_len = 0 if random.random() < .7 else 2**random.randint(6, 12)
+        mutate_depth = random.randint(1, 10)
         command = [
             os.path.join(build_dir, 'src', 'test', 'fuzz', 'fuzz'),
-            "-runs=100000",
+            "-max_total_time=6000",
+            "-reload=0",
```

Comment:
Nice, do you happen to know why they don't only set it to `1` when `-jobs` is not `0`?

Reply:
Oh, I’ve done that :)